tests/dialects/test_snowflake.py

```diff
@@ -1549,6 +1549,81 @@ def test_show(self):
         self.assertTrue(isinstance(users_exp, exp.Show))
         self.assertEqual(users_exp.this, "USERS")
 
+    def show_views(self):
```

Comment:
This isn't a valid test method name, you need the `test_` prefix.

But, ignoring this point, let's just consolidate all these new methods into `test_show` since it already exists. Also, it looks like a lot of these tests are duplicated and basically the only thing that differs is the keyword following `SHOW`. It would be awesome if you could clean this up a bit with a for loop or something, otherwise I'll do it in another PR.